src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2IntegrationTest.java

```diff
@@ -1150,4 +1150,20 @@ private static String keyForVariant( final VariantContext variant ) {
         return String.format("%s:%d-%d %s, %s", variant.getContig(), variant.getStart(), variant.getEnd(), variant.getReference(),
                 variant.getAlternateAlleles().stream().map(Allele::getDisplayString).sorted().collect(Collectors.toList()));
     }
+
+    @Test
+    public void testSachet() throws Exception {
```

Comment:
Can we rename to something about missed duplicates?  And if you want to thank Sachet in a comment, that should be okay.